DRAFT_RELEASE_NOTES.md

```diff
@@ -6,7 +6,8 @@ Data access through [GraphQL](https://graphql.org/) has been deprecated and repl
 # Breaking Changes
 _Note: instructions for adapting to these changes are outlined in the upgrade instructions below._
 
- - _There are no breaking changes in teh 1.11 release._
+ - To improve the development cycle and docker build consistency, we have deprecated the docker_build() and local_resources() functions in the Tilt and enable maven docker build for the docker modules. Follow the instruction in the `Finalizing the Upgrade` to avoid duplicated docker image build.
```

Comment:
A: I don't think this belongs under breaking changes since it doesn't actually break anything and they _could_ continue using their Tiltfile as-is without problems.  We should add something under _**Major Additions**_ and we could have this text live there. E.g.

```md
## Path to Production Alignment
To better align development processes with processes in CI/CD and higher environments, we no longer recommend using Tilt for building and deploying projects.  As such, upgrading projects should consider removing or at least narrowing the scope of their Tiltfile. See _**How to Upgrade**_ for more information.
```

We will likely end up tweaking this section before we actually release to align with our final recommendations, so I wouldn't stress to much about making it perfect.